apps/molgenis-components/src/components/account/MolgenisSession.vue

```diff
@@ -82,7 +93,8 @@ export default defineComponent({
       showChangePasswordForm: false,
       error: null as string | null,
       loading: false,
-      session: {} as ISession,
+      session: { locale: "en", settings: [] } as ISession,
```

Comment:
Session is defined in data but also used as a prop, I think it should be moved to props.

Reply:
in this component it should be a data. In other components it is a prop.  So I don't understand what you think should be done?

Reply:
In Molgenis.vue it is used like this: 
 <MolgenisSession
          v-model="session"
So I think Molgenis.vue should be changed?

Reply:
I tried to remove it in Molgenis.vue and that breaks the application, so I think it should be a prop (maybe with default?) and not data. I'm kind of surprised Vue doesn't give a warning for this.